Resources/docs/README.md

```diff
@@ -28,6 +28,10 @@ Customizing the right control sidebar:
 
 * [Control Sidebar](control_sidebar.md)
 
+Extending webpack-encore:
+
+* [Extend Webpack Encore](extend_webpack_encore.md)
```

Comment:
Could you move the bullet point "Building front-end assets" to this heading as well, so all frontend related docs are at one place?